CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 1.1.0-nullsafety.0
```

Comment:
@hovadur could you please change the README's record once you change the version in pubspec

Reply:
done